clients/filesystem-fuse/src/utils.rs

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// join the parent and name to a path
+pub fn join_file_path(parent: &str, name: &str) -> String {
+    if parent.is_empty() {
+        name.to_string()
+    } else {
+        format!("{}/{}", parent, name)
```

Comment:
is it possible parent ends with '/'?

Reply:
We will handle it in the subsequent PR

Reply:
it's a bug, please handle this in this PR

Reply:
I have made significant changes to filename later on, and my subsequent PR has already addressed it. Can't we leave it for the next PR?"

Reply:
@diqiu50 If you plan to fix this problem in the next PR, Please create an issue and assign to yourself, now.